app/Data/migrations/versions/774ce7027729_comment_like_table.py

```diff
@@ -17,8 +18,17 @@
 
 
 def upgrade():
-    pass
+    op.create_table(
```

Comment:
UserId and CommentId needs to have sa.Foreignkey value to link tables,
and not sure if sa.BOOLEAN is a function
try sa.Boolean instead